deploy/bz1997062-crio-pid-leak/config.yaml

```diff
@@ -0,0 +1,6 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchExpressions:
+  - key: hive.openshift.io/version-major-minor
+    operator: In
+    values: ["4.7", "4.8"]
```

Comment:
According to the BZ we know precisely what versions are impacted:
* 4.7.24 - 4.7.28
* 4.8.5 - 4.8.9 
* Maybe 4.6.44+ if a fix doesn't arrive before then.

So it might be better to do a targeted deployment to those versions using `hive.openshift.io/version-major-minor-patch` instead.

Reply:
@mrbarge done